### ISSUE TITLE
"from Metric import distEuclidean"提示没有该模块

### ISSUE THREAD

**Issue by willard-yuan** [jeromewang-github/computer_vision#1]
Hi, jerome

看你写的一些图像特征，非常感兴趣，你这个是用python3的语法写的？我在测试的时候，提示没有`from Metric import distEuclidean`，谷歌发觉也没有现有的模块中有这样的模块的，请问`Metric`这个模块是出自哪里？

盼复，谢谢。

**Comment by willard-yuan** [comment]
`from Metric import distEuclidean`应该来自`sklearn.metrics.pairwise`吧，将其改成`from sklearn.metrics.pairwise import euclidean_distances as distEuclidean`可以正常运行，这个特征提取的Repo还在开发吗？我有意向加入进来和你一起拓展它。

**Comment by jeromewang-github** [comment]
谢谢你的关注～
当时还有个文件没有传上来，你的替换策略是对的～程序里面还是有不少bug的，然后有些特征提取的策略需要改一改，有些在用的时候已经想到不合理的情况了～
我也觉得这些特征提取的实现还是非常有用的，不过最近不太有时间整这个东西了，得写毕业论文～
有问题欢迎一起交流学习～

**Comment by willard-yuan** [comment]
你现在是应该研二了吧？我觉得可以把这几个常用的特征好好搞一下，python对这几个特征的提取还没有这样的库可用。
